RetailCoder.VBE/UI/SourceControl/SourceControlViewViewModel.cs

```diff
@@ -116,6 +125,27 @@ public void RemoveComponent(VBComponent component)
             }
         }
 
+        public void RenameComponent(VBComponent component, string oldName)
```

Comment:
This is really *reacting* to a component that *was renamed*. `HandleRenamedComponent` would make that clearer.